src/DependencyInjection/Configuration.php

```diff
@@ -37,6 +37,55 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->addDefaultsIfNotSet()
             ->children()
+                ->arrayNode('pdf_rendering')
+                ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('default_orientation')->defaultValue('Portrait')->end()
```

Comment:
```suggestion
                        ->scalarNode('orientation')->defaultValue('Portrait')->end()
```

Could we replace `'Portrait'` with a constant?

Can you add info? I.e. `The default orientation for rendered gift cards`

Reply:
Where would you put that constant ?

Reply:
Some class - not sure of the name of that class though :D